flink-connectors/flink-connector-pulsar/src/test/resources/containers/txnStandalone.conf

```diff
@@ -0,0 +1,980 @@
+#
```

Comment:
Do we need all of these options or could we remove some values that are defaults anyways?
Or is it more meant as a way to protect against changing default values?